util/execdetails/execdetails.go

```diff
@@ -450,14 +451,23 @@ func (crs *CopRuntimeStats) String() string {
 		}
 	}
 
+	var result string
 	if totalTasks == 1 {
-		return fmt.Sprintf("time:%v, loops:%d", procTimes[0], totalIters)
+		result += fmt.Sprintf("time:%v, loops:%d", procTimes[0], totalIters)
+	} else {
+		n := len(procTimes)
+		sort.Slice(procTimes, func(i, j int) bool { return procTimes[i] < procTimes[j] })
+		result += fmt.Sprintf("proc max:%v, min:%v, p80:%v, p95:%v, iters:%v, tasks:%v",
+			procTimes[n-1], procTimes[0], procTimes[n*4/5], procTimes[n*19/20], totalIters, totalTasks)
+	}
+	if crs.copDetails != nil {
+		result += fmt.Sprintf(", total_keys:%v, processed_keys:%v, rocksdb:{delete_skipped_count:%v, "+
```

Comment:
@breeswish, What do you think of these field names?

Reply:
I'm fine with those

Reply:
Do we need any modification about this pr?